mp2-v1/src/query/planner.rs

```diff
@@ -65,7 +65,7 @@ impl<'a, C: ContextProvider> NonExistenceInput<'a, C> {
         }
     }
 
-    pub(crate) async fn find_row_node_for_non_existence(
+    pub async fn find_row_node_for_non_existence(
```

Comment:
Just curious, why do you need to make this function public to use it in DQ? In my mind everything needed for DQ should be provided by `generate_chunks_and_update_tree` method